### PR TITLE
Typings for session/events

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

File: src/session/cookies.ts [hunk 1]
```diff
@@ -7,10 +7,10 @@ const LOCALHOST_DOMAIN: string = 'localhost';
 const LOCALHOST_IP: string     = '127.0.0.1';
 
 export default class Cookies {
-    private cookieJar: any;
+    private _cookieJar: any;
```
Comment (miherlosev) [inline src/session/cookies.ts]: private readonly

Comment (LavrovArtem) [reply src/session/cookies.ts]: https://github.com/DevExpress/testcafe-hammerhead/pull/1940/files#diff-69cf2c774bcce54a35fd29f6b37855f7R64

Comment (miherlosev) [reply src/session/cookies.ts]: Ok

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Pull request merged**